src/test/java/uk/gov/justice/digital/delius/service/AppointmentServiceTest.java

```diff
@@ -26,8 +27,7 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.mockito.ArgumentMatchers.any;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
```

Comment:
IDE settings? I Think we normally go for no wildcard imports

Reply:
will change